docs/search/query-development-guide.md

```diff
@@ -0,0 +1,186 @@
+Hypotheses of user expectations and search behaviour are validated by analysis.  Once validated, different query tools are deployed and tested in order to meet these expectations.  
+
+In cases where links to analysis or relevance testing are missing, this work is either pending or redundant. In cases where links to query tools are missing, this query type is served by the tools used for Concepts, subjects.
+
+
+# [Concepts, subjects](https://docs.wellcomecollection.org/catalogue/search/hypotheses/concepts-subjects.md)
+
+### Analysis conducted
+[Less than 3-word searches](https://docs.wellcomecollection.org/catalogue/search/analysis/less-than-3-word-searches.md)
+
+[Searches with 3 words or more](https://docs.wellcomecollection.org/catalogue/search/analysis/summary-3-plus-word-searches.md)
+
+### Query tools used
```

Comment:
I'm not sure query tools is the write phraseology here. I would consider Elasticsearch to be a tool, or Kibana. These are parts of [Elasticsearche's DSL](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html).

Reply:
I'm also a little confused as to what we mean that we used these tools for the analysis?

Reply:
I agree that between "search," "query" and ways queries are built, the vocabulary space is getting a little congested.  I think referring to Bool boosting as a DSL or AST query is going to make this even more confusing.

The idea was to present what goes on under the hood as a combination of different search strategies depending on user search input. What do you think of changing this from "query tool," to "search tool," or "search strategy?" 

There are 2 different sections: Analysis conducted and Query tools used. Analysis conducted is about how we used data analysis to validate the hypotheses (eg do users search for contributors?)  Query tools used is about the combinations of tools used to serve up the best results.

Reply:
of those options, "search strategy" is the least confusing to me

Reply:
Personally, I don't think i'd find this kind of 'field-first' guide very helpful if I were trying to understand how our queries have developed. 

We're trying to develop new queries by focusing on user intentions first, not fields. Because so many of the individual analyses/tests we've done touch multiple fields, the shift in perspective for this document results in loads of repeated links.